libbeat/docker-compose.yml

```diff
@@ -42,14 +43,15 @@ services:
   proxy_dep:
     image: busybox
     depends_on:
-      elasticsearch:            { condition: service_healthy }
-      elasticsearch_monitoring: { condition: service_healthy }
-      elasticsearchssl:         { condition: service_healthy }
-      logstash:                 { condition: service_healthy }
-      kafka:                    { condition: service_healthy }
-      redis:                    { condition: service_healthy }
-      sredis:                   { condition: service_healthy }
-      kibana:                   { condition: service_healthy }
+      elasticsearch:                  { condition: service_healthy }
+      elasticsearch_kerberos.elastic: { condition: service_healthy }
```

Comment:
Nit: just name this `elasticsearch_kerberos` (unless the `.elastic` suffix is somehow significant)?

Reply:
It is needed because it is part of the ELASTIC Kerberos realm.